.github/workflows/main-workflow.yml

```diff
@@ -14,6 +14,10 @@ jobs:
     uses: ./.github/workflows/update-tailwind-bundle.yml
     secrets: inherit
 
+  update-thrid-party-notices:
```

Comment:
I would include this in automated fixes

Reply:
oh wow just kidding it takes 14 minutes to run?? do we know why it's so slow?

Reply:
It normally takes around 2-3 mintues, I'm not sure why it took so long that time

Reply:
I think it makes sense to put it in there since it's an "automated fix". Moved!